src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocMethodCheck.java

```diff
@@ -137,6 +180,59 @@
  *   &lt;property name="ignoreMethodNamesRegex" value="^foo.*$"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *   public void test1() {} // violation, method is missing javadoc
+ *   public void foo() {} // ok
+ *   public void foobar() {} // ok
+ * }
+ * </pre>
+ *
+ * <p>
+ * To configure the check for ignoring missing javadoc for accessor methods:
+ * </p>
+ * <pre>
+ * &lt;module name="MissingJavadocMethod"&gt;
+ *   &lt;property name="allowMissingPropertyJavadoc" value="true"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *   private String text;
+ *
+ *   public void test() {} // violation, method is missing javadoc
+ *   public String getText() { return text; } // ok
+ *   public void setText(String text) { this.text = text; } // ok
+ * }
+ * </pre>
+ *
+ * <p>
+ * To configure the check with annotations that allow missed documentation:
+ * </p>
+ * <pre>
+ * &lt;module name="MissingJavadocMethod"&gt;
+ *   &lt;property name="allowedAnnotations" value="Override,Deprecated"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *     public void test() {} // violation, method is missing javadoc
+ *     &#64;Override
+ *     public void test1() {} // OK
+ *     &#64;Deprecated
+ *     public void test2() {} // OK
+ *     &#64;SuppressWarnings
+ *     public void test3() {} // violation, method is missing javadoc
+ *     &#47;**
+ *      * Some description here.
+ *      *&#47;
+ *     &#64;SuppressWarnings
+ *     public void test4() {} // OK
```

Comment:
for lines above indentation is 4, but in other examples it is 2.
Please make it 2 here as well to keep all the code consistent

Reply:
Done

Reply:
Done